### PR TITLE
CLDR-15545 Update locales.txt for Meta

### PR DESCRIPTION
**Pull request by AEApple**
CLDR-15545

- Update locales.txt with new languages as requested by Meta
- Rename FB to Meta in Locales.txt

- [x] This PR completes the ticket.

### CODE REVIEW

File: tools/cldr-code/src/main/java/org/unicode/cldr/util/Organization.java [hunk 1]
```diff
@@ -37,6 +36,7 @@ public enum Organization {
     lakota_lc("Lakota LC"),
     lao_dpt("Lao Posts/Telecom??"),
     longnow("The Long Now Foundation", "Long Now", "PanLex", "Utilka Foundation"),
+    meta("Meta"),
```
Comment (srl295) [inline tools/cldr-code/src/main/java/org/unicode/cldr/util/Organization.java:39]: Needs to retain an alias to Facebook.

**Review by srl295 — CHANGES_REQUESTED** [review]
Needs an alias. I’ll add a more specific comment.

**Comment by srl295** [comment]
@AEApple probably should have a (short) procedure somewherer for this process

File: tools/cldr-code/src/main/java/org/unicode/cldr/util/Organization.java (hunk 1 above)
Comment (srl295) [reply tools/cldr-code/src/main/java/org/unicode/cldr/util/Organization.java:39]:
```suggestion
    meta("Meta", "Facebook"),
```

**Review by srl295 — APPROVED** [review]

**Review by pedberg-icu — APPROVED** [review]

**Review by rickmcgowan — APPROVED** [review]
The changed files look reasonable to me. Note I've updated the cldr-reply address list on corp.unicode.org to add Meta, and also retained FB for now.

**Pull request merged**